oci_spec/src/lib.rs

```diff
@@ -465,7 +466,7 @@ pub struct LinuxResources {
     pub freezer: Option<FreezerState>,
 }
 
-#[derive(Serialize, Deserialize, Debug, Clone, Copy)]
+#[derive(Serialize, Deserialize, Debug, Clone, Copy, PartialEq, Eq, Hash)]
```

Comment:
Why do you need to implement the Hash? I'm not seeing this used anywhere. I'm also unsure where you need Eq or PartialEq.

Reply:
You are right. This was leftover from something I was experimenting with. Will clean up.